### DISCUSSION TITLE
How to return consumer result to producer

### DISCUSSION THREAD

**Discussion by xyx361100238** [edenhill/librdkafka#3951] [Q&A]
There is a way to return consumer result to producer？

**Comment by rolandyoung** [comment]
Short answer: No.
Slightly longer answer: Which consumer? At the time a message is produced, there can be many consumers for a topic or none.
If you know the consumer group to which a specific consumer belongs, you can monitor whether it is lagging behind the producer(s) - see https://docs.confluent.io/cloud/current/monitoring/monitor-lag.html.
